src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+function parseSalary(salary) {
+  return parseInt(salary.replace('$', '').replace(',', ''));
```

Comment:
The `parseSalary` function assumes that the salary string will always contain a dollar sign and a comma. Consider adding error handling or validation to ensure the input is in the expected format to prevent potential errors.